code/client/src/components/Pages/Register.js

```diff
@@ -95,18 +84,25 @@ class Register extends React.Component {
                 </form>
               </div>
               <div className="register-right-info">
-                  <h2>Register Now – It’s quick & easy!</h2>
+                  <h1>Register Now – It’s quick & easy!</h1>
                   <h2>By registering you’ll be able to log your trips, add photos, share your travel experiences, rate your donations and view other traveler blogs.</h2>
                   <h2>DoMAD will never sell your personal information.</h2>
               </div>
             </div>
         );
     }
-}
 
-function mapState(state) {
-    const { registering } = state.registration;
-    return { registering };
+
+handleChange = event => {
+  this.setState({
+    [event.target.name]: event.target.value
+  });
+};
+
+handleSubmit = event => {
+  console.log("Submitting");

```

Comment:
If you add event.preventDefault it will stop the browser from going to the page that shows the return message

Reply:
Same within the login page js too